open-reac/src/main/java/com/powsybl/openreac/parameters/input/OpenReacParameters.java

```diff
@@ -35,14 +33,26 @@ public class OpenReacParameters {
 
     private final List<String> variableTwoWindingsTransformers = new ArrayList<>();
 
-    private OpenReacOptimisationObjective objective = OpenReacOptimisationObjective.MIN_GENERATION;
+    // Algo parameters
 
-    private Double objectiveDistance;
+    private OpenReacOptimisationObjective objective = OpenReacOptimisationObjective.MIN_GENERATION;
 
     private OpenReacAmplLogLevel logLevelAmpl = OpenReacAmplLogLevel.INFO;
 
     private OpenReacSolverLogLevel logLevelSolver = OpenReacSolverLogLevel.EVERYTHING;
 
+    private static final String OBJECTIVE_DISTANCE_KEY = "ratio_voltage_target";
+
+    private Double objectiveDistance = 0.5; // between 0 and 100
```

Comment:
Don't provide default value for a `Double` objective distance: it can be null without any issue depending on the objective chosen.